docs/source/nlp/token_classification.rst

```diff
@@ -0,0 +1,277 @@
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.num_samples                 | integer         | Number of samples to use from the dev set, -1 - to use all                                                   |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.batch_size                  | integer         | Dev set batch size                                                                                           |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.name                                | string          | Optimizer to use for training                                                                                |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.lr                                  | float           | Learning rate to use for training                                                                            |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.weight_decay                        | float           | Weight decay to use for training                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.name                          | string          | Warm up schedule                                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| optim.sched.warmup_ratio                  | float           | Warm up ratio                                                                                                |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+Example of the command for training the model:
+
+.. code::
+
+    python token_classification_train.py \
+           model.dataset.data_dir=<PATH_TO_DATA_DIR>  \
+           trainer.max_epochs=<NUM_EPOCHS> \
+           trainer.gpus=[<CHANGE_TO_GPU(s)_YOU_WANT_TO_USE>]
+
+
+Required Arguments for Training
```

Comment:
Similar suggestion to drop the titles of optional and required arguments. You may merge these arguments section.